configuration/configure-automatic-upgrades.adoc

```diff
@@ -1,18 +1,21 @@
 [id="configure-automatic-upgrades"]
-= Configuring automatic upgrades for secured clusters
+= Configuring automatic upgrades for manifest-installed secured clusters
 include::modules/common-attributes.adoc[]
 :context: configure-automatic-upgrades
 
 toc::[]
 
 [role="_abstract"]
-You can automate the upgrade process for each secured cluster and view the upgrade status from the {product-title-short} portal.
+If you installed {product-title} by using the manifest installation method, also known as the `roxctl` CLI method, you can automate the upgrade process for each secured cluster. You can also view the upgrade status from the {product-title-short} portal.
```

Comment:
> also known as the `roxctl` CLI method

we also call it the "legacy installation method" and I think this is how we refer to it in the UI
![Screenshot 2024-12-06 at 10 32 37](https://github.com/user-attachments/assets/2a4ad38e-f014-4f54-8b49-0e6a11c9a6f6)